tools/migrators/lib/tests/grafana/test_transform.py

```diff
@@ -0,0 +1,112 @@
+"""
+Unit tests for the Grafana Service Model transformation logic.
+"""
+
+import unittest
+from unittest.mock import Mock
+
+from lib.grafana.transform import transform_service, validate_component
+from lib.pagerduty.resources.business_service import BusinessService
+from lib.pagerduty.resources.services import TechnicalService
+
+
+class TestTransform(unittest.TestCase):
```

Comment:
let's not use `unittest.TestCase`, to be consistent with the existing tests, and instead just use `pytest` assertions, see for example the [`TestPagerDutyMigrationFiltering`](https://github.com/grafana/oncall/pull/5485/files#diff-11f92d30f99eac1e5e6d08e0a2abb2aff015792b52ace30152d08f7c01fd771fR179) test class